index.html.haml

```diff
@@ -79,6 +78,20 @@ desc: Weld - CDI Reference Implementation
               %th
                 Available Patches
           %tbody
+            %tr
+              %td
+                WildFly 12.0.0.Final
+              %td
+                Weld 3.0.3.Final
+                %span{:class=>"label label-primary"}
+                  CDI 1.2
+                &nbsp;
+                %span{:class=>"label label-info"}
+                  CDI 2.0 preview
```

Comment:
This may be be a bit confusing.
CDI 2.0 is full blown but opt-in, so maybe `CDI 2.0 (opt-in via property)`?

Reply:
Well, officially it is preview and the property is called `ee8.preview.mode`. But I agree that your proposal sounds better. I will also add a link to http://wildfly.org/news/2018/02/28/WildFly12-Final-Released/.